UIKitHomework/UIKitHomework/FoodViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FoodViewController.swift
+//  UIKitHomework
+//
+//  Created by Vera Zaitseva on 30.06.2021.
+//
+
+import UIKit
+
+class FoodViewController: UIViewController {
+    let pizzaButton = UIButton()
+    let sushiButton = UIButton()
```

Comment:
```suggestion
    private let pizzaButton = UIButton()
    private let sushiButton = UIButton()
```
все свойства, которые не вызываются вне скоупа должны быть private